libs/fluid-elements/switch/src/lib/switch.ts

```diff
@@ -193,12 +193,51 @@ export class FluidSwitch extends LitElement {
   }
   private _checked = false;
 
+  /**
+   * Role of the switch.
+   * @private - An internal prop that should not appear in the readme and should
+   * not be set by the outside.
+   */
+  @property({
+    type: String,
+    reflect: true,
+  })
+  role: string = 'switch';
+
+  /**
+   * Aria-checked attribute of the switch.
+   * @private - An internal prop that should not appear in the readme and should
+   * not be set by the outside.
+   */
+  @property({
+    type: String,
+    reflect: true,
+    attribute: 'aria-checked',
+  })
+  ariaChecked: string = 'false';
+
   /** First updated lifecycle */
   firstUpdated(props: Map<string | number | symbol, unknown>): void {
```

Comment:
you can use the `PropertyValues` from lit-element here for the type

Reply:
Thank you :)